access-db.js

```diff
@@ -443,6 +445,74 @@ async function getEmissionsPerVehicle(companyEmail, licensePlateFilter) {
     return emissions;
 }
 
+async function getTripData(contactEmail, licensePlate, cO2Consumed, distance, duration, fuelUsed, upperTimeBound, lowerTimeBound) {
```

Comment:
Since we know that there will likely be more data options later, what if we just accepted a dictionary like  `{ CO2Consumed: true, distance: true, duration: true, fuelUsed: true, upperTimeBound: null, lowerTimeBound: null }`? That way, the method signature would not change frequently.

I'd propose `getTripData(companyEmail, licensePlate, desiredProperties)`

Reply:
ooo smart, I agree. We might have to be careful about SQL injection stuff so ima implement some checks on the array before we process it just to make sure nothing weird ends up in it.